src/stub_launcher.cc

```diff
@@ -442,7 +442,9 @@ StubLauncher::Launch()
         stub_message_queue_.reset();
         parent_message_queue_.reset();
         memory_manager_.reset();
-        WaitForStubProcess();
+        if (stub_pid_ != 0) {
```

Comment:
Can we add a comment here to explain that the check is required because the stub may have been terminated if there's some error occurred - just for future reference.